### DISCUSSION TITLE
Is it possible to remove $U in blocked.html?$S&$U  ?

### DISCUSSION THREAD

**Discussion by Rodvi** [proginosko/LeechBlockNG#451] [Q&A] [answered]
I don't want to see URL of the blocked page ($U) but I need to see the block set number ($S). Because if I do right-click on URL, the context menu will appear with an option to save a page, and this is what I want to avoid.
I tried to replace `blocked.html?$S&$U` by `blocked.html?$S` in the options but this didn't work...

Also it would be nice to additionally enable $K (keyword that triggered the block, if there was one).

**Comment by proginosko** [comment, marked as answer]
The default blocking page expects both $S and $U in its own URL. If you don't want the URL of the blocked page, you'll probably need to create a custom blocking page. But it wouldn't be much work to add an option to remove the hyperlink from the URL. Perhaps post that in the Ideas category to see if there's support for it?

The keyword is already available via $K, but that's only useful for custom blocking pages.
